### PR TITLE
Feature/sar 417 measure table styling

### PR DESCRIPTION
**Pull request by timAmida**
# Related Tickets

- [SAR-417](https://jira.amida-tech.com/browse/SAR-417)

# Other Repos' PR(s) Intended to Work With This PR

- None

# How Things Worked (or Didn't) Before This PR

-Measure table not styled

# How Things Work Now (And How to Test)
- Measure Table Styled
# Readiness

1. [x] This PR passes all automated tests.
2. [x] This PR has no linting errors.
3. [x] This PR's changes to configuration files have been documented in all appropriate places (such as but not limited to README.md), if applicable.

### CODE REVIEW

File: src/components/MeasureResults/MeasureResultsRow.js [hunk 1]
```diff
@@ -8,50 +8,50 @@ import React from 'react';
...
+              onChange={(event) => handleMeasureChange(event)}
+            />
+          </FormGroup>
+        </Grid>
+        <Grid item xs={1}>
+          <Typography variant="caption" className="measure-results-table__data-measure-title">
             {measureResult.label}
           </Typography>
         </Grid>
         <Grid item xs={2}>
```
Comment (anthem123) [inline src/components/MeasureResults/MeasureResultsRow.js:30]: Give the grid items a class and put a `margin: auto 0` to center the text. Important if we are going to have the longer titles in the data table

File: src/components/MeasureResults/MeasureResultsRow.js [hunk 2]
```diff
@@ -8,50 +8,50 @@ import React from 'react';
 function MeasureResultsRow({ measureResult, handleMeasureChange, measureColor }) {
   return (
     <Box>
-      <Divider color="black" />
-      <Grid container justifyContent="space-evenly" direction="row" alignItems="center" spacing={2} sx={{ width: '100%', p: '3px', m: '2px' }}>
+      <Divider color="#78909C" stroke={2} />
+      <Grid container justifyContent="space-evenly" align-items="center" direction="row" spacing={2} sx={{ width: '100%', p: '3px', m: '0px' }}>
```
Comment (anthem123) [inline src/components/MeasureResults/MeasureResultsRow.js:12]: You will want to put these styles in a scss file to be consistent.

File: src/components/MeasureResults/MeasureResultsRow.js (hunk 1 above)
Comment (timAmida) [reply src/components/MeasureResults/MeasureResultsRow.js:30]: moving into class now.

File: src/components/MeasureResults/MeasureResultsRow.js (hunk 2 above)
Comment (jmfadeley) [reply src/components/MeasureResults/MeasureResultsRow.js:12]: `MeasureResultsTable` is doing many of the same things. You can probably `@extend` that.

File: src/context/DatastoreReducer.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ const updateTimestamp = () => {
 
 const createLabel = (measure, info) => {
   if (info[measure]) {
-    return info[measure].displayLabel;
+    return info[measure].title;
```
Comment (jmfadeley) [inline src/context/DatastoreReducer.js:21]: I think you want to combine these two. `${info[measure].displayLabel} - ${info[measure].title}`.

File: src/test/data/DemoData.js [hunk 4]
```diff
@@ -2940,7 +2940,7 @@ export const trendList = [
 
 export const infoObject = {
   composite: {
-    title: 'Composite',
+    title: 'Composite Score',
```
Comment (jmfadeley) [inline src/test/data/DemoData.js:2943]: That's not going to reflect in the real data. It'll just read `Composite - Composite`. @anthem123 would have to change what is being sent.

File: src/components/MeasureResults/MeasureResultsRow.js (hunk 2 above)
Comment (timAmida) [reply src/components/MeasureResults/MeasureResultsRow.js:12]: Can do. let me repush

File: src/test/data/DemoData.js (hunk 4 above)
Comment (timAmida) [reply src/test/data/DemoData.js:2943]: SO should i get @anthem123 to show me?

File: src/components/MeasureResults/MeasureResultsTable.js [hunk 5]
```diff
@@ -33,7 +35,13 @@ function MeasureResultsTable({ currentResults, handleMeasureChange, colorMapping
...
       <Grid container item className="measure-results-table__header-section">
         <Grid item className="measure-results-table__title-align-small">
-          <Typography className="measure-results-table__title">
+          <Typography className="measure-results-table__measure-title">View</Typography>
+          <ToolTip title={viewTip}>
+            <HelpIcon className="measure-results-table__help-icon" />
+          </ToolTip>
+        </Grid>
+        <Grid item className="measure-results-table__title-align-small">
+          <Typography className="measure-results-table__measure-title">
```
Comment (jmfadeley) [inline src/components/MeasureResults/MeasureResultsTable.js:44]: I really don't see any reason for this new class at all.

File: src/components/ChartContainer/D3Container.js [hunk 6]
```diff
@@ -147,7 +147,9 @@ function D3Container({ dashboardState, dashboardActions, store }) {
   const handleMeasureChange = (event) => {
     let newSelectedMeasures;
     if (event.target.checked) {
-      newSelectedMeasures = selectedMeasures.concat(event.target.value);
+      newSelectedMeasures = event.target.value === 'all'
+        ? store.currentResults.map((result) => result.measure)
+        : selectedMeasures.concat(event.target.value);
       setSelectedMeasures(newSelectedMeasures);
     } else {
       newSelectedMeasures = selectedMeasures.filter(
```
Comment (anthem123) [inline src/components/ChartContainer/D3Container.js:155]: Currently when I select the top checkbox when everything is selected, nothing happens. I'm assuming everything is supposed to be unchecked. If that is the case just throw in this on line 154 between the if and the else

`else if (event.target.value === 'all') {
      newSelectedMeasures = selectedMeasures.filter(
        (result) => result === event.target.value,
      );
      setSelectedMeasures(newSelectedMeasures);
    }`

**Review by anthem123 — APPROVED** [review]
Stylish and functional, what a combination. Approved

**Review by eawest2 — APPROVED** [review]
LGTM

**Pull request merged**